pandas/tests/test_frame.py

```diff
@@ -5129,6 +5129,11 @@ def test_operators(self):
         df = DataFrame({'a': ['a', None, 'b']})
         assert_frame_equal(df + df, DataFrame({'a': ['aa', np.nan, 'bb']}))
 
+        df = DataFrame()
+        self.assertTrue((df + df).equals(DataFrame()))
```

Comment:
pls add the issue number as a comment.

I would also add a couple of other test, things like

```
df = DataFrame(columns=['A'])
self.assertTrue((df+df).equals(df)
```
and add for an index, but no columns